build.sh

```diff
@@ -106,6 +112,8 @@ defaults write "$PWD/$onDemainDaemonFileName" Label -string "$onDemandDaemonIden
 defaults write "$PWD/$preferenceFileName" UpdateInProgressTitle -string "$updateInProgressTitle"
 defaults write "$PWD/$preferenceFileName" UpdateInProgressMessage -string "$updateInProgressMessage"
 defaults write "$PWD/$preferenceFileName" LoginAfterUpdatesInProgressMessage -string "$loginAfterUpdatesInProgressMessage"
+defaults write "$PWD/$preferenceFileName" notificationActionButtonText -string "$notificationActionButtonText"
```

Comment:
Change the key that is being written to NotificationActionButtonText (uppercase N) to stay consistent with the other keys in the plist. The variable can keep the lowercase n.

Reply:
Makes complete sense. Sorry for not spotting it! :)